### ISSUE TITLE
African and Native Slaves must be consistent regarding founding new cities and working as scouts.

### ISSUE THREAD

**Issue by abmpicoli2** [Religion-And-Revolution-Extended/Mod#7]
Discussion at forum https://forums.civfanatics.com/threads/religion-and-revolution-bugs-and-todos.449958/page-121#post-15023890

**Comment by abmpicoli2** [comment]
Does comments appear at slack as well??

**Comment by abmpicoli** [comment]
I have checked the current code, after a release build. The only professions allowed are raw materials outside. as expected...
![image](https://user-images.githubusercontent.com/1733170/45641406-d0236080-ba8b-11e8-9c2f-c96f2d91b01c.png)

Also detected that the player can't found new colonies with it...

**Comment by abmpicoli** [comment]
Closing.
Tested if the AI founds cities with slaves. With this two savegames, it was demonstrated that AI can't found new cities with slaves: At the game start, I added two slaves for Hans Egede (AI) . Ended my turn, and it couldn't find a city. At the next turn, I've added a free colonist. With that, he found a colony.

[slave scenarios.zip](https://github.com/Religion-And-Revolution-Extended/Mod/files/2389762/slave.scenarios.zip)
